torch_xla/test/test_utils.py

```diff
@@ -101,11 +101,17 @@ def get_summary_writer(logdir):
     Instance of Tensorboard SummaryWriter.
   """
   if logdir:
-    from tensorboardX import SummaryWriter
-    writer = SummaryWriter(log_dir=logdir)
-    write_to_summary(
-        writer, 0, dict_to_write={'TensorboardStartTimestamp': time.time()})
-    return writer
+    try:
+      from tensorboardX import SummaryWriter
+      writer = SummaryWriter(log_dir=logdir)
+      write_to_summary(
+          writer, 0, dict_to_write={'TensorboardStartTimestamp': time.time()})
+      return writer
+    except Exception as e:
+      print("Failed to initialize Tensorboard SummaryWriter: %s" % str(e),
+            flush=True)
+      return None
```

Comment:
We'll end up calling `None.method` in our test scripts. Let's return `from torch.utils.tensorboard import SummaryWriter` this summary writer instead if the logdir is GCS prefix. wdyt

Reply:
that makes sense to me. Only problem is we can't seem to import `from torch.utils.tensorboard import SummaryWriter` during our pytorch xla training script (based on `test_train_mp_mnist.py`: https://b.corp.google.com/issues/192592433#comment3